### PR TITLE
provide more driver info in logs

### LINKED ISSUE #131: provide more driver info in logs
**Is your feature request related to a problem?/Why is this needed**

**Describe the solution you'd like in detail**

could refer to https://github.com/kubernetes-csi/csi-driver-smb/blob/a9a2e47b1790b18e722e9d0b4d413c4e8a520507/pkg/smb/smb.go#L54-L58

```
[pod/csi-smb-controller-5dd89b9bf6-vbbzq/smb] DRIVER INFORMATION:
[pod/csi-smb-controller-5dd89b9bf6-vbbzq/smb] -------------------
[pod/csi-smb-controller-5dd89b9bf6-vbbzq/smb] Build Date: "2021-01-09T16:38:17Z"
[pod/csi-smb-controller-5dd89b9bf6-vbbzq/smb] Compiler: gc
[pod/csi-smb-controller-5dd89b9bf6-vbbzq/smb] Driver Name: smb.csi.k8s.io
[pod/csi-smb-controller-5dd89b9bf6-vbbzq/smb] Driver Version: e2e-c091d36a1d9ee8d6026cb06a165bbe49110afcb5
[pod/csi-smb-controller-5dd89b9bf6-vbbzq/smb] Git Commit: c091d36a1d9ee8d6026cb06a165bbe49110afcb5
[pod/csi-smb-controller-5dd89b9bf6-vbbzq/smb] Go Version: go1.15.5
[pod/csi-smb-controller-5dd89b9bf6-vbbzq/smb] Platform: linux/amd64
```

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by boddumanohar**
**What type of PR is this?**
/kind feature

**What this PR does / why we need it**:
The changes in this PR provides more information about the driver when the driver is started.
 
**Which issue(s) this PR fixes**:

Fixes #131 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
none
```

### CODE REVIEW

File: pkg/nfs/nfs.go [hunk 1]
```diff
@@ -98,6 +98,13 @@ func NewNodeServer(n *Driver, mounter mount.Interface) *NodeServer {
 }
 
 func (n *Driver) Run(testMode bool) {
+
```
Comment (andyzhangx) [inline pkg/nfs/nfs.go:101]: remove empty line

File: pkg/nfs/version.go [hunk 2]
```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2019 The Kubernetes Authors.
```
Comment (andyzhangx) [inline pkg/nfs/version.go:2]: 2021

**Review by andyzhangx — COMMENTED** [review]
/ok-to-test

**Comment by andyzhangx** [comment]
could you fill in the missing fields in this PR? thanks.
```
[pod/csi-nfs-controller-b565969cc-7fcmm/nfs] DRIVER INFORMATION:
[pod/csi-nfs-controller-b565969cc-7fcmm/nfs] -------------------
[pod/csi-nfs-controller-b565969cc-7fcmm/nfs] Build Date: N/A
[pod/csi-nfs-controller-b565969cc-7fcmm/nfs] Compiler: gc
[pod/csi-nfs-controller-b565969cc-7fcmm/nfs] Driver Name: nfs.csi.k8s.io
[pod/csi-nfs-controller-b565969cc-7fcmm/nfs] Driver Version: N/A
[pod/csi-nfs-controller-b565969cc-7fcmm/nfs] Git Commit: N/A
[pod/csi-nfs-controller-b565969cc-7fcmm/nfs] Go Version: go1.15.7
[pod/csi-nfs-controller-b565969cc-7fcmm/nfs] Platform: linux/amd64
```

**Comment by boddumanohar** [comment]
@andyzhangx  fixed

**Review by andyzhangx — APPROVED** [review]
/lgtm
/approve

**Pull request merged**